lib-ee/emqx_license/test/emqx_license_installer_SUITE.erl

```diff
@@ -0,0 +1,77 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_license_installer_SUITE).
+
+-compile(nowarn_export_all).
+-compile(export_all).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("common_test/include/ct.hrl").
+-include_lib("snabbkaffe/include/snabbkaffe.hrl").
+
+all() ->
+    emqx_common_test_helpers:all(?MODULE).
+
+init_per_suite(Config) ->
+    _ = application:load(emqx_conf),
+    emqx_common_test_helpers:start_apps([emqx_license], fun set_special_configs/1),
+    Config.
+
+end_per_suite(_) ->
+    emqx_common_test_helpers:stop_apps([emqx_license]),
+    ok.
+
+init_per_testcase(_Case, Config) ->
+    {ok, _} = emqx_cluster_rpc:start_link(node(), emqx_cluster_rpc, 1000),
+    Config.
+
+end_per_testcase(_Case, _Config) ->
+    ok.
+
+set_special_configs(emqx_license) ->
+    Config = #{type => file,
+               value => emqx_license_test_lib:default_license()},
+    emqx_config:put([license], Config);
+
+set_special_configs(_) -> ok.
+
+%%------------------------------------------------------------------------------
+%% Tests
+%%------------------------------------------------------------------------------
+
+t_update(_Config) ->
+    ?check_trace(
+       begin
+           ?wait_async_action(
+              begin
+                Pid0 = spawn_link(fun() -> receive exit -> ok end end),
+                register(installer_test, Pid0),
+
+                {ok, _} = emqx_license_installer:start_link(
+                           installer_test,
+                           ?MODULE,
+                           10,
+                           fun() -> ok end),
```

Comment:
Small suggestion: if this callback sent a `emqx_license_installer_called` to the test pid, maybe the test could be simplified by not using `?check_trace`.